src/plugin.js

```diff
@@ -815,12 +815,16 @@ for (let i = 0; i < _readOnly.length; i++) {
  * Check if the Flash tech is currently supported.
  *
  * @return {boolean}
- *          - True if the flash tech is supported.
- *          - False otherwise.
+ *          - True always as the latest versions of Chrome and Firefox automatically
+ *            blocks flash by default.
  */
 Flash.isSupported = function() {
-  return Flash.version()[0] >= 10;
-  // return swfobject.hasFlashPlayerVersion('10');
+  // for IE
+  if (videojs.browser.IE_VERSION) {
+    return Flash.version()[0] >= 10;
+  }
+  // for other browsers
+  return true;
```

Comment:
after some discussion with @forbesjo, we decided this is a bit too risky as it is. Instead, we should whitelist desktop chrome and desktop safari and return true just for those browsers and keep the version check for other browsers.